### PR TITLE
Update view cell doc [ci skip]

### LINKED ISSUE #3668: Documentation Error
Under View Cells guide, under the section "Cell Parameters"
In the last example, the parameters passed to recentPosts() method wasn't accessed correctly.

### PR DESCRIPTION
**Pull request by paulbalandan**
**Description**
Fixes #3668 .

~Along the way, I found a teeny error in `intro/index` so I fixed that too.~

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPdocs
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/intro/index.rst [hunk 1]
```diff
@@ -17,9 +17,9 @@ or completely replaced to make the system work the way you need it to.
 In short, CodeIgniter is the malleable framework that tries to provide
 the tools you need while staying out of the way.
 
-***********************
-Who is CodeIgniter For?
-***********************
+************************
+For Whom is CodeIgniter?
```
Comment (michalsn) [inline user_guide_src/source/intro/index.rst:21]: English is not my first language but that sounds really formal and a bit unnatural to me. Can we go back to the initial form?

**Review by michalsn — CHANGES_REQUESTED** [review]

File: user_guide_src/source/intro/index.rst (hunk 1 above)
Comment (paulbalandan) [reply user_guide_src/source/intro/index.rst:21]: Considering the word order, the correction should have been: "Whom is CodeIgniter For?" But this construction is too informal.

The original construction of "Who ... for" is grammatically incorrect as "who" would be referring to CodeIgniter and "for" is misplaced. The intended usage of the title is asking for the intended users so it should have been "whom".

Comment (michalsn) [reply user_guide_src/source/intro/index.rst:21]: I will have to leave this decision to @MGatner since I'm not qualified enough 😅

Comment (MGatner) [reply user_guide_src/source/intro/index.rst:21]: The emendation is grammatically correct... and very stiff. But using "who" where one should use "whom" is a bit egregious (in my book). What if we ditch this and go with something else altogether, like "Is CodeIgniter For You?" or "CodeIgniter's Target Audience" or "Who Uses CodeIgniter?" - something both keeping with the User Guide's colloquial tone without butchering the already poor state of the English language.

**Comment by paulbalandan** [comment]
Extracted the intro/index changes to a separate PR, #3681 . Never expected this to grow that much. 😂

Also, sorry for two topics on one PR. I've learned my lesson there. :)

**Review by MGatner — APPROVED** [review]

**Pull request merged**